lib/common/results.c

```diff
@@ -436,7 +438,7 @@ crm_errno2exit(int rc)
 const char *
 bz2_strerror(int rc)
 {
-    /* http://www.bzip.org/1.0.3/html/err-handling.html */
+    // See ftp://sources.redhat.com/pub/bzip2/docs/manual_3.html#SEC17
     switch (rc) {
```

Comment:
https://www.sourceware.org/pub/bzip2/docs/manual_3.html#SEC17
also works

Reply:
Although the newest (just one point release higher),
yet still authentic may be
https://web.archive.org/web/20180723131348/http://www.bzip.org/1.0.3/html/err-handling.html
(the domain in a suspicious state now)

Reply:
Ah, so this is the ultimate end point I think:
https://web.archive.org/web/20180727053619/http://www.bzip.org:80/1.0.5/bzip2-manual-1.0.5.html#err-handling

Reply:
It's rather that previous official home was abandoned in favour of
a dedicated domain (which may have been subsequently squated recently).

I'd say it's still better to have most up-to-date authoritative
documentation linked (even if through proxy) rather than some
ancient one.  (There was no documentation like that exposed for
`1.0.6` so next best is `1.0.5`.)

Alternatively, `bzip2-devel` Fedora/EL RPM package carries
`/usr/share/doc/bzip2-devel/manual.html`.